react-app/src/components/GovernanceInfoPanel/GovernanceInfoPanel.tsx

```diff
@@ -35,9 +58,56 @@ const GovernanceInfoPanel: React.FC = () => {
     );
   }
 
+  const {
+    deposit: { minDeposit, maxDepositPeriod },
+    tally: { quorum, threshold, vetoThreshold },
+    voting: { votingPeriod },
+  } = govParams;
+
   return (
-    <Paper className={cn("")}>
-      <pre>{JSON.stringify(govParams, null, 4)}</pre>
+    <Paper>
+      <div className={cn("flex", "items-center", "mb-5", "gap-[10px]")}>
+        <Icon icon={IconType.Info} className={cn("w-6", "h-6")} />
+        <h2 className={cn("text-lg", "leading-[18px]", "font-bold")}>
+          <LocalizedText messageID="GovernanceInfoPanel.governanceInfo" />
+        </h2>
+      </div>
+      <div
+        className={cn("flex", "flex-col", "sm:flex-row", "gap-3", "sm:gap-16")}
+      >
+        <InfoField
+          labelMessageID="GovernanceInfoPanel.minDeposit"
+          value={
+            convertBigNumberToFixedPointString(minDeposit.amount) +
+            " " +
+            minDeposit.denom.toUpperCase()
```

Comment:
maybe ``${convertBigNumberToFixedPointString(minDeposit.amount)} ${minDeposit.denom.toUpperCase()}`` to prevent adding a space string?

Reply:
looks weird.... do you mean like this?

```suggestion
            <>
              {convertBigNumberToFixedPointString(minDeposit.amount)}{" "}
              {minDeposit.denom.toUpperCase()}
            </>
```

Reply:
like this?
```suggestion
           <>
           {`${convertBigNumberToFixedPointString(minDeposit.amount)} ${minDeposit.denom.toUpperCase()}`}
           </>
```